src/dotnet/APIView/APIViewWeb/Pages/Shared/_Layout.cshtml

```diff
@@ -21,27 +21,23 @@
     <header>
         <nav class="navbar navbar-expand-sm navbar-toggleable-sm navbar-light bg-light border-bottom mb-3">
             <div class="container">
-                <a class="navbar-brand" asp-area="" asp-page="/Index">APIView</a>
```

Comment:
Why?

Reply:
I didn't think having a link to reviews here was necessary since a separate one also exists, and I don't want to link to anything through the APIView logo if the user isn't logged in. I could put the link back for logged in users, but I didn't think it was necessary to add the logic for minimal functionality added

Reply:
Sounds good.